api.R

```diff
@@ -11,91 +12,130 @@ source("generics.R")
 # Read AWS creds from Docker secrets
 if (file.exists("/run/secrets/ENV_FILE")) {
   readRenviron("/run/secrets/ENV_FILE")
-} else {
+} else if (file.exists("secrets/ENV_FILE")) {
   readRenviron("secrets/ENV_FILE")
+} else {
+  readRenviron(".env")
 }
-readRenviron(".env")
 
 # Get the model run attributes at runtime from env vars
 dvc_bucket <- Sys.getenv("AWS_S3_DVC_BUCKET")
 run_bucket <- Sys.getenv("AWS_S3_MODEL_BUCKET")
-run_id <- Sys.getenv("AWS_S3_MODEL_RUN_ID")
-run_year <- Sys.getenv("AWS_S3_MODEL_YEAR")
```

Comment:
I don't think the run year is necessary since we can always derive it from the first four characters of the run ID, do you agree?

Reply:
Cool, I made the year configurable for each model in abe540391c05b750507e5086fedbf27b31c49d51.